service/build.gradle.kts

```diff
@@ -60,6 +60,8 @@ dependencies {
         Dependencies.Reflections,
         Dependencies.BouncyCastle,
         Dependencies.BouncyCastleProvider,
+        Dependencies.OpenApi.WebFluxSupport,
```

Comment:
I'm not familiar with OpenApi.  Looks like a good add and I'm interested to see documentation output.
👍